dandiapi/api/views/dandiset.py

```diff
@@ -262,6 +263,31 @@ def destroy(self, request, dandiset__pk):
         dandiset.delete()
         return Response(None, status=status.HTTP_204_NO_CONTENT)
 
+    @swagger_auto_schema(
+        methods=['POST'],
+        manual_parameters=[DANDISET_PK_PARAM],
+        request_body=no_body,
+        responses={
+            200: 'Dandiset unembargoing dispatched',
+            400: 'Dandiset not embargoed',
+        },
+        operation_summary='Unembargo a dandiset.',
+        operation_description=(
+            'Unembargo an embargoed dandiset. Only permitted for owners and admins'
+            '. If the embargo status is OPEN or UNEMBARGOING, an HTTP 400 is returned.'
+        ),
+    )
+    @action(methods=['POST'], detail=True)
+    @method_decorator(permission_required_or_403('owner', (Dandiset, 'pk', 'dandiset__pk')))
+    def unembargo(self, request, dandiset__pk):
```

Comment:
Can you add this view to the `test_embargo_visibility` parametrization?

Reply:
739f00c